flake/nixosModules/profile-basic.nix

```
@@ -106,6 +106,7 @@
 
         cron.enable = true;
         fail2ban.enable = true;
+        netdata.enable = true;
```

I have this config here:

```suggestion
        services.netdata = {
          enable = true;
          config.ml.enabled = true;
        };
```
Also adding `systemd.services.netdata.path = [pkgs.jq];` was needed.

---

As of netdata version v1.32.0, ML is enable by default.  We're good to go.